.gitignore

```diff
@@ -21,3 +21,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea
+.
```

Comment:
do you need all of these types in you gitignore?
.idea/*
*.iml
target/*
would be enough?